src/test/java/com/pubnub/contract/objectV2/step/WhenSteps.kt

```diff
@@ -0,0 +1,85 @@
+package com.pubnub.contract.objectV2.step
+
+import com.pubnub.contract.objectV2.state.GetAllUUIDMetadataState
+import com.pubnub.contract.objectV2.state.GetUUIDMetadataState
+import com.pubnub.contract.objectV2.state.RemoveUUIDMetadataState
+import com.pubnub.contract.objectV2.state.SetUUIDMetadataState
+import com.pubnub.contract.state.World
+import io.cucumber.java.en.When
+
+
+class WhenSteps(
+    private val getUUIDMetadataState: GetUUIDMetadataState,
+    private val setUUIDMetadataState: SetUUIDMetadataState,
+    private val removeUUIDMetadataState: RemoveUUIDMetadataState,
+    private val getAllUUIDMetadataState: GetAllUUIDMetadataState,
+    private val world: World
+) {
+
+
+    @When("I get the UUID metadata")
+    fun I_get_the_UUID_metadata() {
+        getUUIDMetadataState.result = world.pubnub.getUUIDMetadata()
+            .uuid(getUUIDMetadataState.id)
+            .includeCustom(false)
```

Comment:
My choice would be to not touch `includeCustom` at all. The step doesn't say anything about it